packages/vscode-extension/src/utilities/packageManager.ts

```diff
@@ -37,25 +70,39 @@ export async function resolvePackageManager(): Promise<PackageManagerInfo | unde
       } as const)
     );
 
-    const files = await fs.readdir(workspace);
+    const getDirFilesSortedByModificationDate = async (dir: string) => {
```

Comment:
can we have it defined somewhere outside of the scope? this seems like a utility function and doesn't require to be in this function scope